src/main/js/controls/Graph/helpers/helpers.js

```diff
@@ -419,6 +452,45 @@ const createVGridHandler = (gridSVG, axis, style, config) => {
         .classed(style, true)
         .call(translateVerticalGrid(axis, config));
 };
+/**
+ * calculates the character limit of label with respect to axis size
+ *
+ * @private
+ * @param {string} config - config object derived from input JSON
+ * @returns {object} character limit for both and y-axis
+ */
+const getAxesLabelCharacterLimits = (config) => ({
+    xAxisLimit: (3 * getXAxisWidth(config)) / 22,
```

Comment:
Should we mention here which width requires how many characters?

Reply:
[Updated](https://github.com/cerner/carbon-graphs/commit/005b577ae3f43708560ca2241645bf3f370269be#diff-e136c9760b04608293a9d079cc909772R40) documentation.

Actually, For every ≈7 pixel(depends on character),  we can accommodate one uppercase character. As width of x-axis is long(around 1400px), if we assign 8px for each character we will get some small space left in x-axis. In order to fill that space i tried to assign  less than 8(22/3 i.e ≈7.33)px for each character in x-axis. That is the reason why i used different numbers to divide axis length for both x-axis and y-axis.

Reply:
if we are targeting 7.33 pixels per character, is there a reason that we cannot make it a little simpler and just go with 
```javascript
xAxisLimit: getXAxisWidth(config) / 7.333
```
It might be slightly less precise but it would be unlikely to affect the outcome.

Reply:
[fixed](https://github.com/cerner/carbon-graphs/pull/242/commits/9a24873cfed7035a62bd84bc0a1eedf230f426d7#diff-608946a83d2c753fa2af8bbb05dc72d2R470). Changed pixels from fraction to decimal in xAxisLimit Calculation.